test/tools/fuzzing/fuzz.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import csv
+import os
+from pathlib import Path
+import threading
+import time
+import shlex
+import subprocess
+import sys
+import signal
+
+def find_in_path(paths, file_matcher, debug_file_name):
```

Comment:
Maybe leave a comment to say this is the same impl as in run_tests.py so someone can update both if it ever needs to change